sig-contributor-experience/projects.md

```diff
@@ -0,0 +1,73 @@
+# Projects and Goals
+
+This is a list of the projects and goals currently underway with Contributor Experience. Please submit a PR if you are adding your project to this list. To introduce a new project, attend a weekly meeting or drop a note to us on the mailing list - details can be found on our [README](README.md). 
+
+Want to contribute? Take a look at this list and the "future" at the bottom. Thank you to all of our contributors for your hard work!
+
+## SIG Planning
+
+Project | Owner(s)/Lead(s) | Description | Q1, Q2, Later
+---|---|---|---
+[Charter](https://github.com/kubernetes/community/pull/1847) | SIG Leads | Create our first Charter iteration | Q1
+Goals and Projects | SIG Leads | Create projects.md | Q1
```

Comment:
Referring Goals and Projects for this item looks little awkward :-) but I don't have any problem with it. Though the title of this doc says "Projects and Goals" and so let's mentioned as such instead of reversing it?